$OpenBSD$
Disable "download ffmpeg" buttons in GUI, since ffmpeg is provided by packages.
--- src/FFmpeg.cpp.orig	Thu Jul 10 02:51:43 2014
+++ src/FFmpeg.cpp	Thu Jul 10 02:52:46 2014
@@ -649,12 +649,6 @@ class FindFFmpegDialog : public wxDialog (public)
       }
    }
 
-   void OnDownload(wxCommandEvent & WXUNUSED(event))
-   {
-      wxString page = wxT("http://manual.audacityteam.org/o/man/faq_installation_and_plug_ins.html#ffdown");
-      ::OpenInDefaultBrowser(page);
-   }
-
    wxString GetLibPath()
    {
       return mLibPath.GetFullPath();
@@ -675,7 +669,6 @@ class FindFFmpegDialog : public wxDialog (public)
 
 BEGIN_EVENT_TABLE(FindFFmpegDialog, wxDialog)
    EVT_BUTTON(ID_FFMPEG_BROWSE, FindFFmpegDialog::OnBrowse)
-   EVT_BUTTON(ID_FFMPEG_DLOAD,  FindFFmpegDialog::OnDownload)
 END_EVENT_TABLE()
 
 
